src/i18n/fr_FR.ts

```diff
@@ -0,0 +1,821 @@
+                {
+                    caption: 'Gynécologie',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Urologie',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Traumatologie',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Neurologie',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Neurochirurgie',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Cardiologie',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Chirurgie cardiaque',
+                    credits: 70_000,
+                    coins: 15,
+                    duration: '7 jours',
+                },
+                        ],
+            levelcost: ['1.-20. 19.000 crédits / 11 pièces'],
+            maxBuildings: 'Aucune limite',
+            maxLevel: 20,
+            special: 'Le chef des finances et les administrateurs peuvent construire et agrandir les hôpitaux grance à la banque de l\'alliance',
+            startPersonnel: 0,
+            startVehicles: [],
+        },
+        {
+            caption: 'Station d\'hélicoptère',
+            color: '#e7ad2f',
+            coins: 50,
+            credits: 1_000_000,
+            extensions: [],
+            levelcost: [],
+            maxBuildings: 'see specials',
+            maxLevel: 0,
+            special:
+                'Nombre de stations max : nombre de bâtiments divisé par 25.',
+            startPersonnel: 0,
+            startVehicles: [],
+            maxBuildingsFunction: (buildingsAmountTotal: number): number =>
+                buildingsAmountTotal < 125
+                    ? 4
+                    : Math.floor(buildingsAmountTotal / 25),
+        },
+        {
+            caption: 'Gendarmerie',
+            color: '#007700',
+            coins: 35,
+            credits: 100_000,
+            extensions: [
+                ...new Array(10).fill({
+                    caption: 'Cellule de prison',
+                    credits: 25_000,
+                    coins: 5,
+                    duration: '7 jours',
+                }),
+            ],
+            levelcost: ['1. 10.000', '2. 50.000', '3.-39. 100.000'],
+            maxBuildings: '1200 postes de police/gendarmerie',
+            maxLevel: 14,

```

Comment:
16

Reply:
This building have only 14 level, not 16 like the others